kotlin/internal/jvm/BUILD

```diff
@@ -33,3 +33,13 @@ bzl_library(
         "@build_bazel_rules_android//android",
     ],
 )
+
+load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
```

Comment:
@nkoroste I think we need to include `bazel_skylib` in the release archive as a release dependency.

Reply:
Not sure how releases work to be honest so I would differ to your expertise